ovos_backend_client/pairing.py

```diff
@@ -38,6 +38,9 @@ def has_been_paired():
     Returns:
         bool: True if ever paired with backend (not factory reset)
     """
+    if Configuration()[“server”].get(“backend_type”) not in [“selene”, "personal"]:
```

Comment:
It might be safer to make this a “const”, in case we ever want to support additional backends that pair